core/src/test/java/hudson/util/Digester2Test.java

```diff
@@ -30,28 +30,38 @@
 
 import java.io.FileNotFoundException;
 import java.net.ConnectException;
+import java.util.Locale;
 
 import static org.hamcrest.CoreMatchers.containsString;
+import static org.hamcrest.MatcherAssert.assertThat;
 
-public class Digester2Security2147Test {
+public class Digester2Test {
 
     private static final String RESOURCE_FILE_NAME = "Digester2Security2147TestData.xml";
 
     @Issue("SECURITY-2147")
     @Test
     public void testProtection() throws Exception {
+        Locale defaultLocale = Locale.getDefault();
         try {
-            new Digester2().parse(Digester2Security2147Test.class.getResourceAsStream(RESOURCE_FILE_NAME));
-            Assert.fail("expected exception");
-        } catch (SAXParseException ex) {
-            Assert.assertThat(ex.getMessage(), containsString("DOCTYPE is disallowed"));
+            // otherwise it's not passing on machine not running english as OS language
```

Comment:
Had to update this test as it was failing on my machine working with French default :)